packages/docs/src/stories/components/input.test.stories.ts

```diff
@@ -883,7 +884,7 @@ export const Samples = {
               }}
               class="ml-4 scale-[1.714] inline-flex items-center sd-interactive"
             >
-              <sd-icon name="system/minus-circle"></sd-icon>
+              <sd-icon name="system/minus-circle" label="Decrease button"></sd-icon>
```

Comment:
Please remove this whole example. We now handle this completely different, as stepUp and stepDown are now with steppers  integrated: https://solid-design-system.github.io/solid/fix_sd-input-a11y-issues/?path=/docs/templates-input--docs – instead adapt the following and add it to the "Type" screenshots:

```
<sd-input
    label="Spin Buttons"
    type="number"
    min="0"
    max="100"
    spin-buttons
  ></sd-input>
```